app/clients/sms/twilio.py

```diff
@@ -240,18 +248,23 @@ def translate_delivery_status(
             error_code = parsed_dict['ErrorCode'][0]
 
             if error_code in self.twilio_error_code_map:
-                notify_delivery_status = self.twilio_error_code_map[error_code]
+                notify_delivery_status: TwilioStatus = self.twilio_error_code_map[error_code]
             else:
-                notify_delivery_status = self.twilio_notify_status_map[twilio_delivery_status]
+                self.logger.warning('Unacounted for Twilio Error code: %s', error_code)
```

Comment:
Can we include the `MessageSid` or something on these logs so we have a way to trace messages back if they have unexpected errors?

Reply:
Added, fixed the typo, and added type hinting for the logger.